packages/block-library/src/template-part/edit/placeholder.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * WordPress dependencies
+ */
+import { useEntityBlockEditor, EntityProvider } from '@wordpress/core-data';
+import { __ } from '@wordpress/i18n';
+import { BlockPreview } from '@wordpress/block-editor';
+import { useState, useCallback } from '@wordpress/element';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { cleanForSlug } from '@wordpress/editor';
```

Comment:
Isn't the core block library already coupled to WordPress?

Where could we move this utility function?

Reply:
Currently, the block library does not depend on the @wordpress/editor module directly although we still have two indirect dependencies via the store that we are removing.
I think some blocks are WordPress specific or rely on the WordPress API I guess it may make sense to separate the block library into WordPress specific blocks and generic blocks in the future.
But for now, I guess a possible package for cleanForSlug would be @wordpress/url.

Reply:
Done 👍